idea_town/experiments/models.py

```diff
@@ -18,6 +18,8 @@ class Experiment(models.Model):
     description = models.TextField()
     measurements = MarkupField(blank=True, default='',
                                default_markup_type='plain')
+
+    order = models.IntegerField(default=0)
```

Comment:
Might also be interesting to add [a model Meta class](https://docs.djangoproject.com/en/1.9/topics/db/models/#meta-options) here [with an `ordering` default for this field](https://docs.djangoproject.com/en/1.9/ref/models/options/#ordering).